matminer/featurizers/site.py

```diff
@@ -413,24 +413,32 @@ def from_preset(preset, **kwargs):
                 else:
                     op_types[k + 1] = ["wt"]
 
-            return CrystalNNFingerprint(op_types, **kwargs)
+            return CrystalNNFingerprint(op_types, chem_info=None, **kwargs)
 
         else:
             raise RuntimeError('preset "{}" is not supported in '
                                'CrystalNNFingerprint'.format(preset))
 
-    def __init__(self, op_types, **kwargs):
+    def __init__(self, op_types, chem_info=None, **kwargs):
         """
         Initialize the CrystalSiteFingerprint. Use the from_preset() function to
```

Comment:
This is not your mistake, but might as well fix it - change ``CrystalSiteFingerprint`` to ``CrystalNNFingerprint``.